fineract-core/src/main/java/org/apache/fineract/infrastructure/core/config/FineractProperties.java

```diff
@@ -190,6 +192,30 @@ public static class FineractEventsProperties {
         private FineractExternalEventsProperties external;
     }
 
+    @Getter
+    @Setter
+    public static class FineractTaskExecutor {
+
+        private FineractEventTaskExecutor eventTaskExecutor;
```

Comment:
I don't think this is in the right place. The property organization was horizontal until this point and now you're making it vertical. 

The event related configuration should be with the rest of the event properties.

Reply:
replaced under the event properties